test/parallel/test-process-versions.js

```diff
@@ -14,6 +18,8 @@ const expected_keys = [
   'napi',
   'llhttp',
   'uvwasi',
+  'undici',
```

Comment:
You might want to surround all the `undici` tests in this file with `if-else`s that check that `process.config.variables.node_builtin_shareable_builtins` contains the `'deps/undici/undici.js'` string because if it's not present, it would mean that the build was configured with an externalized undici. For that case, the test should check that `process.versions` does not contain `undici`.

Reply:
done